native/cocos/physics/physx/PhysXWorld.h

```diff
@@ -125,17 +125,24 @@ class PhysXWorld final : virtual public IPhysicsWorld {
     float getFixedTimeStep() const override { return _fixedTimeStep; }
     void setFixedTimeStep(float fixedTimeStep) override { _fixedTimeStep = fixedTimeStep; }
 
+#if CC_USE_GEOMETRY_RENDERER
     virtual void setDebugDrawFlags(EPhysicsDrawFlags flags) override;
     virtual EPhysicsDrawFlags getDebugDrawFlags() override;
 
     virtual void setDebugDrawConstraintSize(float size) override;
     virtual float getDebugDrawConstraintSize() override;
```

Comment:
In `else` branch, there are not `virtual` prefix for overrode method. So let's keep remove `virtual` here.